jstz_node/src/services/accounts.rs

```diff
@@ -29,11 +30,63 @@ async fn nonce(
     Ok(HttpResponse::Ok().json(nonce))
 }
 
+#[get("/{address}/kv/{key:.*}")]
+async fn kv(
+    rollup_client: Data<RollupClient>,
+    path: Path<(String, String)>,
+) -> Result<impl Responder> {
+    let (address, key) = path.into_inner();
+
+    let storage_key = if key.is_empty() {
+        format!("/jstz_kv/{}", address)
+    } else {
+        format!("/jstz_kv/{}/{}", address, key)
+    };
+
+    let value = rollup_client.get_value(&storage_key).await?;
+
+    println!("value: {:?}", value);
+
+    let value = match value {
+        Some(value) => bincode::deserialize::<KvValue>(&value)
+            .map_err(|_| anyhow!("Failed to deserialize account"))?,
+        None => return Ok(HttpResponse::NotFound().finish()),
+    };
+
+    Ok(HttpResponse::Ok().json(value))
+}
+
+#[get("/{address}/kv/{key}/subkeys")]
+async fn kv_subkeys(
+    rollup_client: Data<RollupClient>,
+    path: Path<(String, String)>,
+) -> Result<impl Responder> {
+    let (address, key) = path.into_inner();
+
+    let storage_key = if key == "EMPTY" {
+        format!("/jstz_kv/{}", address)
+    } else {
+        format!("/jstz_kv/{}/{}", address, key)
+    };
```

Comment:
I'm not a massive fan of this. I think it is good for listing the root keys, but it feels messy 🤔

Reply:
Additionally keys like `TABLE/KEY` don't get matched by this endpoint

Reply:
Perhaps doing `/kv?key={}` would be best for us and `/kv/subkeys?key={}` 🤔